python/pfs/drp/stella/measureFiberNorms.py

```diff
@@ -391,6 +392,31 @@ def measureFiberNorms(self, spectra: Dict[int, PfsArm], visitList: Iterable[int]
 
         return PfsFiberNorms(identity, fiberId, wavelength, values, fiberProfilesHash, model, header)
 
+    def plotFiberNorms(
+        self,
+        fiberNorms: PfsFiberNorms,
+        pfsConfig: PfsConfig,
+        visitList: Iterable[int],
+        arm: str,
+    ) -> "Figure":
+        """Plot fiber normalization values
+
+        Parameters
+        ----------
+        fiberNorms : `pfs.datamodel.pfsFiberNorms.PfsFiberNorms`
+            Fiber normalization values
+        pfsConfig : `pfs.datamodel.PfsConfig`
+            Configuration for the PFS system
+
+        Returns
+        -------
+        fig : `matplotlib.figure.Figure`
+            Figure containing the plot.
+        """
+        fig, axes = plotFiberNorms(fiberNorms, pfsConfig, lower=self.config.plotLower, upper=self.config.plotUpper)
+        axes.set_title(f"Fiber normalization for arm={arm}\nvisits: {','.join(map(str, visitList))}")
+        return fig
+
```

Comment:
Why is this in the "Formatting" commit?

Reply:
It looks like I screwed up the rebase, which also reintroduced the flake errors I had fixed. I'll clean this up.